e2e/pages/launch_vscode.page.js

```diff
@@ -0,0 +1,45 @@
+// launch_vscode.page.js
+const { _electron: electron } = require('playwright');
+const path = require('path');
+const fs = require('fs');
+
+class LaunchVSCodePage {
```

Comment:
If we're going to use this file as the main class for handling the interaction with vscode, maybe we can name the class just "VSCode".